source/simulator/parameters.cc

```diff
@@ -1327,7 +1327,7 @@ namespace aspect
     // preconditioner
     prm.enter_subsection ("Material model");
     {
-      prm.declare_entry ("Material averaging", "none",
+      prm.declare_entry ("Material averaging", "project to Q1 only viscosity",
```

Comment:
I have to think about if this is the right thing to do, because it silently changes the averaging even for models that use AMG preconditioner. I also have to check the cookbooks that discuss averaging to make sure they do use the correct averaging and do not contradict this new default value.